website/src/components/Chat/ChatConversation.tsx

```diff
@@ -58,9 +59,21 @@ export const ChatConversation = memo(function ChatConversation({ chatId, getConf
         sampling_parameters,
       };
 
-      const assistant_message: InferenceMessage = await post(API_ROUTES.CREATE_ASSISTANT_MESSAGE, {
-        arg: assistant_arg,
-      });
+      let assistant_message: InferenceMessage;
+      try {
+        assistant_message = await post(API_ROUTES.CREATE_ASSISTANT_MESSAGE, {
+          arg: assistant_arg,
+        });
+      } catch (e) {
+        if (e instanceof OasstError) {
```

Comment:
This always is `OasstError` since we already transformed the error, the check to make TS happy